libraries/config/src/chain_config.cpp

```diff
@@ -48,12 +48,9 @@ decltype(ChainConfig::predefined_) const ChainConfig::predefined_([] {
     cfg.final_chain.state.eth_chain_config.dao_fork_block = state_api::BlockNumberNIL;
     cfg.final_chain.state.execution_options.disable_nonce_check = true;
     cfg.final_chain.state.execution_options.disable_gas_fee = true;
-    addr_t root_node_addr("de2b1203d72d3549ee2f733b00b2789414c7cea5");
```

Comment:
dont we need to change also genesis dag block sender from few lines above ? `"sender": "de2b1203d72d3549ee2f733b00b2789414c7cea5"`

Reply:
There could be any random address. And it is changing nothing as I see

Reply:
I would say lets remove that particular account from everywhere, where it is not needed so it is not confusing

Reply:
I replaced it with new random address